README.md

```diff
@@ -224,7 +222,13 @@ obtained from the following command run in the repo's root folder:
 git rev-parse HEAD
 ```
 
-### Automatic Deployment with Deno Deploy
+Refer to these guides for using Docker to deploy Deno to specific platforms:
+
+- [Amazon Lightsail](https://deno.land/manual/advanced/deploying_deno/aws_lightsail)
+- [Digital Ocean](https://deno.land/manual/advanced/deploying_deno/digital_ocean)
+- [Google Cloud Run](https://deno.land/manual/advanced/deploying_deno/google_cloud_run)
+
+## Deploy to Deno Deploy
```

Comment:
I think we should move the "Deploy to Deno Deploy" before the Docker section. WDYT?